faust/sensors/monitor.py

```diff
@@ -184,6 +197,10 @@ def __init__(self,
 
         self.metric_counts = Counter()
 
+        self.tp_committed_offsets = {}
+        self.tp_read_offsets = {}
+        self.tp_end_offsets = {}
```

Comment:
Should these be `collections.Counter`?

Reply:
Probably not because these don't count. They are gauges.